recipe_scrapers/aldi.py

```diff
@@ -0,0 +1,97 @@
+# mypy: allow-untyped-defs
+import re
+
+from ._abstract import AbstractScraper
+from ._exceptions import ElementNotFoundInHtml
+from ._utils import get_minutes, get_yields
+
+
+class Aldi(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "aldi.com.au"
+
+    def site_name(self):
+        return "Aldi"
+
+    def author(self):
+        return self.soup.find("meta", {"name": "author"}).get("content")
+
+    def title(self):
+        return self.soup.find("h1").text
+
+    def category(self):
+        title = self.soup.select("a.tab-nav--link.dropdown--list--link.m-active")[
+            0
+        ].text
+        recipe_position = title.find(" Recipe")
+        return title[:recipe_position]
+
+    def prep_time(self):
+        return get_minutes(self._get_value(re.compile("prep", re.IGNORECASE)))
+
+    def cook_time(self):
+        return get_minutes(self._get_value(re.compile("cook", re.IGNORECASE)))
+
+    def total_time(self):
+        total_time = 0
+
+        try:
+            total_time += self.prep_time()
+        except ElementNotFoundInHtml:
+            pass
+
+        try:
+            total_time += self.cook_time()
+        except ElementNotFoundInHtml:
+            pass
+
+        return total_time
```

Comment:
```suggestion
        return (self.prep_time() or 0) + (self.cook_time() or 0)
```
simplified by using the `or` operator to handle the possible none values. let me know if there was additional reasoning for the original code that required this work around!